Dockerfile

```diff
@@ -35,4 +35,5 @@ RUN git checkout $MENDER_VERSION
 ENV mender_version $MENDER_VERSION
 COPY mender-deb-package /usr/local/bin/
 COPY debian/* debian/
+RUN cp support/mender.service debian/mender-client.service
```

Comment:
Are `.service` files in `debian` auto-handled?

Also I'm wondering, does this automatically handle stopping the service when upgrading or removing? That's one thing that has been bugging me, that when you remove the package, the daemon still runs afterwards. Maybe you could have a look at it while you're at it.

Reply:
Related to the above.

Yes, it is handled automatically. For an update, for example, it will just stop, upgrade, start. For an install it will enable and start. For a remove stop + remove.

This is the case already for upstream Debian package but we were avoiding this by manually overriding the systemd targets in `rules`. We had it like this previously because we didn't have a way to configure the client.